vite.config.ts

```diff
@@ -28,6 +28,28 @@ export default createAppConfig({
 						if (id.includes('vue-material-design-icon')) {
 							return 'icons'
 						}
+						if (id.toLowerCase().includes('album')) {
+							return 'albums'
+						}
+						if (id.toLowerCase().includes('folder')) {
+							return 'folders'
+						}
+						if (id.toLowerCase().includes('place')) {
+							return 'places'
+						}
+						if (id.toLowerCase().includes('tag')) {
+							return 'tags'
+						}
+						if (id.toLowerCase().includes('face')) {
+							return 'faces'
+						}
+						if (id.toLowerCase().includes('sidebar') || id.toLowerCase().includes('tab') || id.toLowerCase().includes('leaflet')) {
+							return 'photos-sidebar'
+						}
```

Comment:
This is not quite good I would argue because you mix here also internal modules, e.g. if there is a node module with "tag" inside the name you will group it in "tags" which will make the chunking useless.